app/main.ipynb

```diff
@@ -100,18 +127,31 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
-   "outputs": [],
    "source": [
-    "# write your code here"
+    "array = np.array([0, 0.23, 3.62, 5, 4.5, 5.46, 6.46, 8, 10], dtype=np.float16)\n",
+    "\n",
+    "array[[0, 2, -1]]"
```

Comment:
The array is correctly created with float numbers, and multi-indexing is used to access specific elements. Ensure that the array length is between 5 and 10 as required by the task.